.github/semantic.yml

```diff
@@ -0,0 +1,5 @@
+# Semantic Commit bot: https://github.com/Ezard/semantic-prs
+
+# Always validate the PR title, and ignore the commits
+titleOnly: true
+allowMergeCommits: true
```

Comment:
@nicor88, I think we need `allowMergeCommits` as we have Squash and Merge as default